tutorials/packaging/create-your-first-snap/create-your-first-snap.md

```diff
@@ -50,11 +50,17 @@ Survey
 ## Getting started
 Duration: 1:00
 
-This tutorial is based on Ubuntu (16.04 LTS or greater) and the `snapcraft` APT package.
+### Setup a development environment
```

Comment:
I don't think an additional title is necessary as it comes directly after *Getting started*.